viewer/model/ImageDocument.cpp

```diff
@@ -63,7 +71,22 @@ ImageDocument::ImageDocument(QObject * parent)
   , image_(createDefaultImage())
   , watcher_(nullptr)
   , comparisonWatcher_(nullptr)
-{}
+{
+  init();
+}
+
+void ImageDocument::init()
+{
+  thumbnailsAvailable_ = false;
```

Comment:
ImageDocument is certainly not the right place for all this since we don't need to do this per image tab. Probably should go into a Settings object or so. I can change that after merge though, and maybe search for a nice place for the install button.